src/components/listitems/ListItemTransaction.tsx

```diff
@@ -115,7 +115,7 @@ export const ListItemTransaction = ({
           </View>
         )}
         <View style={IOStyles.flex}>
-          <H6 color={theme["textBody-default"]}>{title}</H6>
+          <H5 numberOfLines={2}>{title}</H5>
```

Comment:
```suggestion
          <H5 numberOfLines={2} color={theme["textBody-default"]}>{title}</H5>
```

just noticed this one

Reply:
You're right! In order to proceed with this adjustment, I added that color as AllowedColor into `H5` component and everything is addressed in this commit: 82c3a886a2e29f7ca3a1c8533b8c6fc9c86e9df4

Reply:
As suggested by @dmnplb, with this commit 584f43422485a8d6d1a2ef3e1a81a58b6cb71795 i switched from H5 component to `LabelSmall`